.github/workflows/push-to-npm-gpr.yml

```diff
@@ -28,3 +28,15 @@ jobs:
         run: npm publish --access public mitre-saf-v*.tgz
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      # Setup .npmrc file to publish to GitHub Package Registry
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: 'https://npm.pkg.github.com'
```

Comment:
if we're not using it to build but only publish, do we need to set the node-version attribute?  should we do it just to be on the safe side?

Reply:
Yes you needed it see this url that also covers the scope: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages

Reply:
I will add it, but I don't believe it is necessary as this is set up on the runner and does not haver any interaction with our code or dependencies, so letting the default node-version being used is fair.

Reply:
To be clear, I do not think it uses the same runner as the first invocation of the setup-node action.  From the readme: "If not supplied, the node version from PATH will be used. However, it is recommended to always specify Node.js version and don't rely on the system one."  Consequently, I think we should explicitly set the node version here as well.